package.json

```diff
@@ -85,21 +80,16 @@
     "@percy/playwright": "1.0.7",
     "@playwright/experimental-ct-react": "1.50.1",
     "@playwright/test": "1.50.1",
-    "@rollup/plugin-node-resolve": "15.1.0",
     "@types/node": "20.4.8",
     "@types/react": "18.3.11",
     "@typescript-eslint/eslint-plugin": "5.62.0",
     "@typescript-eslint/parser": "5.62.0",
     "@vitest/coverage-c8": "0.29.8",
-    "autoprefixer": "10.4.20",
```

Comment:
is this definitely not used by our web components for applying vendor prefixes to their CSS?

Reply:
I can't see this being used in our repo, outside of pie-docs. I've also compared `dist/index.js` diffs, the CSS of the Storybook deployment for a sample component, as well as Percy code diffs and I don't see anything to suggest it's being used 👍🏼

Reply:
Might need a ticket to fix this potentially, as I think it should be used (if it's not atm). Happy for this to be just created  as a ticket to spike re-adding it though if it's not being used currently.

Reply:
Created as part of DSW-2913 👍🏼